repos/system_upgrade/el8toel9/actors/mariadbcheck/libraries/mariadbcheck.py

```diff
@@ -0,0 +1,53 @@
+from leapp import reporting
+from leapp.libraries.common.rpms import has_package
+from leapp.libraries.stdlib import api
+from leapp.models import InstalledRedHatSignedRPM
+
+# Summary for postgresql-server report
+report_server_inst_summary = (
+    'MariaDB server component will be upgraded. Since RHEL-9 includes'
+    ' MariaDB server 10.5 by default, which is incompatible with 10.3'
+    ' included in RHEL-8, it is necessary to proceed with additional steps'
+    ' for the complete upgrade of the MariaDB data.'
+)
+
+report_server_inst_hint = (
+    'Back up your data before proceeding with the upgrade'
+    ' and follow steps in the documentation section "Migrating to a RHEL 9 version of MariaDB"'
+    ' after the upgrade.'
+)
+
+# Link URL for postgresql-server report
+report_server_inst_link_url = 'https://access.redhat.com/articles/6743671'  # noqa: E501; pylint: disable=line-too-long
```

Comment:
```suggestion
report_server_inst_link_url = 'https://access.redhat.com/articles/6743671' 
```
It's not too long (120) but this has only 75